website/content/en/docs/faq.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "FAQs"
+linkTitle: "FAQs"
+weight: 90
+---
+## General
+
+### How does a provisioner decide to manage a particular node?
```

Comment:
This is covered in concepts. I'm a bit confused about when we would put something in FAQs vs somewhere else. I am on board with FAQs in general, I just want to avoid scenarios where our docs disagree or drift against itself.